utils/cobrautil/templates/yaml_flag_printer.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package templates
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/spf13/pflag"
+)
+
+type YamlFlagPrinter struct {
+	out       io.Writer
+	wrapLimit uint
+}
+
+func NewYamlFlagPrinter(out io.Writer, wrapLimit uint) *YamlFlagPrinter {
+	return &YamlFlagPrinter{
+		out:       out,
+		wrapLimit: wrapLimit,
+	}
+}
+
+func (p *YamlFlagPrinter) PrintHelpFlag(f *pflag.Flag) {
+	formatBuf := new(bytes.Buffer)
+	writeYamlFlag(formatBuf, f)
+
+	wrappedStr := formatBuf.String()
+	flagAndUsage := strings.Split(formatBuf.String(), "\n")
+	flagStr := flagAndUsage[len(flagAndUsage)-1]
```

Comment:
`flagStr` could be moved inside if - it's not used anywhere else.

Reply:
Inlined.